src/main/java/core/basesyntax/Constants.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public class Constants {
+    public static final int DEFAULT_RADIUS = 10;
+    public static final String DEFAULT_COLOR = "white";
```

Comment:
remove this class, move constants to figureSupplier
white - don't hardcode, use enum